core/chains/evm/client/rpc_client.go

```diff
@@ -183,8 +155,82 @@ func NewRPCClient(
 	return r
 }
 
+func (r *RpcClient) SubscribeToHeads(ctx context.Context) (ch <-chan *evmtypes.Head, sub commontypes.Subscription, err error) {
```

Comment:
Why do we need a new method? Should not we rename just `SubscribeNewHead`?

Reply:
I used the naming defined in the MultiNode design document under RpcClient: https://docs.google.com/document/d/1nvgelqk4xbznRy8zdy_KHeEjFjgKkqhSUGXSlDze7nU/edit

Reply:
Yes, but why we need to introduce a new method? Should we rename existing one?